namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlService.scala

```diff
@@ -133,6 +163,12 @@ class HttpControlService(storage: DtabStore) extends Service[Request, Response]
       handlePostDtab(ns, req)
     case (DtabUri(Some(ns)), Method.Delete) =>
       handleDeleteDtab(ns)
+    case (BindUri(ns, path), Method.Get) =>
+      handleGetBind(ns, path)
+    case (AddrUri(ns, path), Method.Get) =>
+      handleGetAddr(ns, path)
+    case (DelegateUri(ns, path), Method.Get) =>
+      handleGetDelegate(ns, path)
```

Comment:
would be nice to support polling requests as an alternative to streaming responses.  perhaps require `?stream=true`?